### PR TITLE
modified footer and implimented visual breakup

### PR DESCRIPTION
**Pull request by Bibhuti05**
WhatsApp Image

### CODE REVIEW

File: src/component/DigitalBG/digitalBg.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import styles from './digitalbg.module.css';
+import VisualBreakup from '@component/common/VisualBreakup/VisualBreakup';
+import MetalBadge from '@component/common/MetalBadge/MetalBadge';
+
+
+const defaultData = {
```
Comment (arijeet-mukherjee) [inline src/component/DigitalBG/digitalBg.tsx:7]: this should come from data.json file

File: src/component/ProtectYourself/ProtectYourself.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import styles from './protectYourself.module.css';
+import VisualBreakup from '@component/common/VisualBreakup/VisualBreakup';
+
+
+const defaultData = {
```
Comment (arijeet-mukherjee) [inline src/component/ProtectYourself/ProtectYourself.tsx:6]: All static data should come from data.json

Comment (Bibhuti05) [reply src/component/ProtectYourself/ProtectYourself.tsx:6]: Screenshot from 2024-07-10 10-11-59

Comment (Bibhuti05) [reply src/component/ProtectYourself/ProtectYourself.tsx:6]: image

File: src/component/DigitalBG/digitalBg.tsx (hunk 1 above)
Comment (Bibhuti05) [reply src/component/DigitalBG/digitalBg.tsx]: image

**Review by arijeet-mukherjee — APPROVED** [review]
LGTM

**Pull request merged**